src/compass/s1_geo_stack.py

```diff
@@ -0,0 +1,468 @@
+#!/usr/bin/env python
+import argparse
+import datetime
+import glob
+import os
+import time
+from collections import defaultdict
+
+import isce3
+import journal
+import numpy as np
+import pandas as pd
+import yaml
+from s1reader.s1_orbit import get_orbit_file_from_dir, parse_safe_filename
+from s1reader.s1_reader import load_bursts
+from shapely import geometry
+
+from compass.utils import helpers
+from compass.utils.geo_grid import get_point_epsg
+
+
+def create_parser():
+    parser = argparse.ArgumentParser(
+        description='S1-A/B geocoded CSLC stack processor.',
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+    parser.add_argument('-s', '--slc-dir', required=True,
+                        help='Directory containing the S1-A/B SLCs (zip files)')
+    parser.add_argument('-d', '--dem-file', required=True,
+                        help='File path to DEM to use for processing.')
+    parser.add_argument('-o', '--orbit-dir', default=None,
+                        help='Directory with orbit files. If None, downloads orbit files')
+    parser.add_argument('-w', '--working-dir', dest='work_dir', default='stack',
+                        help='Directory to store intermediate and final results')
+    parser.add_argument('-sd', '--start-date', help='Start date of the stack to process')
+    parser.add_argument('-ed', '--end-date', help='End date of the stack to process')
+    parser.add_argument('-b', '--burst-id', nargs='+', default=None,
+                        help='List of burst IDs to process. If None, all the burst IDs '
+                             'in the reference date are processed. (default: None)')
```

Comment:
what is the reference date? I don't see it in the list of input arguments. Actually I don't know why would we need reference date at all?

Reply:
Removed the bad reference to reference date